src/components/themed/MainButton.tsx

```diff
@@ -40,7 +40,7 @@ interface Props {
  * A stand-alone button to perform the primary action in a modal or scene.
  */
 export function MainButton(props: Props) {
-  const { children, disabled = false, label, marginRem = [0, 1, 0, 1], onPress, type = 'primary', paddingRem, layout, spinner = false } = props
+  const { children, disabled = false, label, marginRem = 0, onPress, type = 'primary', paddingRem, layout, spinner = false } = props
```

Comment:
Optional: Because you are passing `marginRem = 0` as the default, none of the `MainButton` components will get extra tappable area. If you let this just be `undefined`, then your padding stretch will kick in.